src/components/budget-expiry-notification/data/expiryThresholds.js

```diff
@@ -0,0 +1,50 @@
+import sanitizeHTML from 'sanitize-html';
+import parse from 'html-react-parser';
+import { PLAN_EXPIRY_VARIANTS } from './constants';
+
+const expiryThresholds = {
+  60: ({ date }) => ({
+    notificationTemplate: {
+      title: `Your organization’s plan expires ${date}}`,
```

Comment:
[inform] This repo now supports i18n; one of the steps on the PR description checklist is to ensure new English strings are marked for translation via `react-intl`.

See `defineMessages` ([docs](https://formatjs.io/docs/react-intl/api/#definemessagesdefinemessage)), `FormattedMessage` ([docs](https://formatjs.io/docs/react-intl/components/#formattedmessage)), and `useIntl` ([docs](https://formatjs.io/docs/react-intl/api/#useintl-hook)), imported from `@edx/frontend-platform/i18n`).

(This repo differs from Admin Portal in that `frontend-app-admin-portal` has not yet migrated its existing English strings to `react-intl`.)